### ISSUE TITLE
React data table component- Sprint 5.1 508 Issue  

### ISSUE THREAD

**Issue by JanellC** [US-EPA-CAMD/PI-1#1843]
## Required Fix 

- [x] RF-1

- [x] RF-2

- [x] RF-3

## Context 

See audit report: https://app.zenhub.com/files/287570343/40248587-8c30-4ebc-82f8-dc7975abaf35/download

| Issue # | Issue Title | Issue Description |
|---|---|---|
| RF-1 | Table   lacks programmatic name | A   data table is present but lacks a programmatic name. This issue is a   violation of Section 508 and WCAG 2.0 Success Criterion 1.3.1. |
| RF-2 | Duplicate   ID values exist | There   are at least 2 elements in the DOM that have the same id. This issue is a   violation of WCAG 2.0 Success Criterion 4.1.1 |
| RF-3 | The   State of a Page Element is Missing | Elements   on the page that have a state or property are not communicating the state or   property to assistive technology. This issue is a violation of WCAG 2.0   Success Criterion 4.1.2 |

**Comment by aprematta** [comment]
RF 1 and 2 Pass.

RF 3 is almost there but I did find an issue where the aria sort is not set correctly.

Aria sort is being picked up by ANDI, however there's an issue if I click back and forth between the column headers. To reproduce:

1. Pick 2 column headers next to each other
2. Inspect the elements so you can see the html
3. Click the first column header
4. Click the second column header
5. Repeat steps 3 and 4 a bunch of times and look how the aria sort is being set

Basically in this scenario you are setting the sort to ascending all the time for each column header. However, every other time the aria sort is going to descending, so there is a mismatch between the value and what's on screen.

**Comment by aprematta** [comment]
![Sort.JPG](https://images.zenhubusercontent.com/5f4e4654f75e20d725e0639d/eaddbbbb-ff96-4f40-8081-610673d1f18b)

**Comment by aprematta** [comment]
Final issue resolved closing this ticket
